agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -212,6 +212,23 @@ class EvaluationScenarioOutput(EmbeddedModel):
     variant_id: str
     variant_output: str
 
+# TODO: This should be removed and replaced with EvaluationDB
+# Keppeing it for now for backwards compatibility
+class BulkEvaluationDB(Model):
+    app: AppDB = Reference(key_name="app")
+    organization: OrganizationDB = Reference(key_name="organization")
+    user: UserDB = Reference(key_name="user")
+    status: str
+    evaluation_type: List[str]
+    evaluation_type_settings: EvaluationTypeSettings
+    variants: List[ObjectId]
```

Comment:
Why are we saving the evaluation of multiple variants in the same object? We want to enable the user to run the eval for multiple variants from the same command. However there is no need to save these results in the same evaluation document.  (disregard this comment in case the goal was just to use for human a/b testing)